jobs/JGLOBAL_FORECAST

```diff
@@ -79,6 +90,8 @@ fi
 if [[ "${DO_AERO_FCST}" == "YES" ]]; then
   YMD="${PDY}" HH="${cyc}" declare_from_tmpl -rx \
     COMOUT_CHEM_HISTORY:COM_CHEM_HISTORY_TMPL
+  YMD="${PDY}" HH="${cyc}" RUN="${rCDUMP}" declare_from_tmpl -rx \
+    COMIN_TRACER_RESTART:COM_ATMOS_RESTART_TMPL
```

Comment:
`COMIN_TRACER_RESTART` is only being declared when `DO_AERO_FCST == "YES"`, but `forecast_postdet.sh` is looking for this variable in all cases.